### DISCUSSION TITLE
How to append 1000 new rows to a big excel file?

### DISCUSSION THREAD

**Discussion by Adityak4201** [exceljs/exceljs#2466] [Q&A]
I have a big excel file. How do I write large data to a excel file without reading it synchronously? Is it also possible that I don't have to read complete file?

**Comment by zurmokeeper** [comment]
@Adityak4201 How big is this large file, how many sheets and how many rows are there?

**Comment by Adityak4201** [comment]
@zurmokeeper this file has already 100K records and has one sheet.

**Comment by zurmokeeper** [comment]
@Adityak4201 Is it convenient to provide this excel file? I don't have a good solution for now, but I can think of a good solution when I have time.
